platform-sdk/swirlds-platform-core/src/main/java/module-info.java

```diff
@@ -143,6 +143,7 @@
     requires com.swirlds.virtualmap;
     requires com.fasterxml.jackson.core;
     requires com.fasterxml.jackson.dataformat.yaml;
+    requires com.google.common;
```

Comment:
This is going to get a hard push back, adding a 3rd party dependency.

Reply:
Yep, definitely not gonna fly.

Reply:
This isn't really new in the codebase, though it is in platform. The guava `Hasher` is quite handy and i see Services already use it. Changed the implementation to use the more familiar `HashBuilder`.